src/actions/actions/linode-boot/__init__.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python
+######################################################################
+# Cloud Routes Bridge
+# -------------------------------------------------------------------
+# Actions Module
+######################################################################
+
+import requests
+import time
+
+
+def action(**kwargs):
+    ''' This method is called to action a reaction '''
+    redata = kwargs['redata']
+    jdata = kwargs['jdata']
+    logger = kwargs['logger']
+    run = True
+    # Check for Trigger
+    if redata['trigger'] > jdata['failcount']:
+        run = False
+
+    # Check for lastrun
+    checktime = time.time() - float(redata['lastrun'])
+    if checktime < redata['frequency']:
+        run = False
+
+    if redata['data']['call_on'] not in jdata['check']['status']:
+        run = False
+
+    if run:
+        return call_linode(redata, jdata, logger)
+    else:
+        return None
+
+
+def call_linode(redata, jdata, logger):
+    ''' Perform actual call '''
+    url = 'https://api.linode.com/'
+    params = {
+        "api_action": "linode.reboot",
+        "LinodeID": int(redata['data']['linode_id']),
+        "api_key": str(redata['data']['api_key'])
+    }
+    try:
+        req = requests.post(
+            url, timeout=3.0, data=params, verify=True)
+    except:
+        return False
+    if req.status_code >= 200 and req.status_code < 300:
+        if len(req.json['ERRORARRAY']) > 0:
+            try:
+                error_message = str(req.json()['ERRORARRAY'][0]['ERRORMESSAGE'])
```

Comment:
actioner_1      | 2015-01-06 19:24:56,570 - actions.actioner - ERROR - Got error when attempting to run reaction linode-boot for monitor a3c548b1-42ce-45b5-b199-2362f201aa0a
actioner_1      | 2015-01-06 19:24:56,571 - actions.actioner - ERROR - 'instancemethod' object has no attribute '__getitem__'

I'm thinking maybe I didn't get a JSON message from linode? I did this by using an invalid node id (old)